### PR TITLE
Renamed Function to Runner

### PR DESCRIPTION
**Pull request by calebjclark**

### CODE REVIEW

File: specification/datastore/DatastoreApis.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ export const DatastoreApiSchemas = {
       adminSignature: signatureValidation
         .optional()
         .describe('A signature from the admin Identity'),
-      adminFunction: z.object({
+      adminRunner: z.object({
```
Comment (blakebyrnes) [inline specification/datastore/DatastoreApis.ts]: This should still be admin"Function". Its about a js function in the datastore

File: specification/datastore/DatastoreApis.ts [hunk 2]
```diff
@@ -56,7 +56,7 @@ export const DatastoreApiSchemas = {
...
         ownerType: z
           .enum(['table', 'crawler', 'function', 'datastore'])
```
Comment (blakebyrnes) [inline specification/datastore/DatastoreApis.ts]: this line should changed function to runner

File: specification/datastore/DatastoreApis.ts [hunk 3]
```diff
@@ -262,7 +262,7 @@ export const DatastoreApiSchemas = {
     }),
     result: z.object({}),
   },
-  'Datastore.createInMemoryFunction': {
+  'Datastore.createInMemoryRunner': {
```
Comment (blakebyrnes) [inline specification/datastore/DatastoreApis.ts:265]: Why is this an api? Don't we always create a virtual table with results? Is this just for exploring the db?

File: specification/datastore/DatastoreApis.ts (hunk 1 above)
Comment (calebjclark) [reply specification/datastore/DatastoreApis.ts]: Yes, fixed.

File: specification/datastore/DatastoreApis.ts (hunk 2 above)
Comment (calebjclark) [reply specification/datastore/DatastoreApis.ts]: Fixed

File: specification/datastore/DatastoreApis.ts (hunk 3 above)
Comment (calebjclark) [reply specification/datastore/DatastoreApis.ts:265]: This is needed when you're querying locally. It must be created before it can be queried.

**Pull request merged**